web/migrations/0052_voicechatroom_voicechatparticipant.py

```
@@ -0,0 +1,56 @@
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("web", "0051_teamgoalmember_completion_image_and_more"),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)